docs/modules/ROOT/pages/user-guide/configuration.adoc

```diff
@@ -103,35 +103,6 @@ Since 1.8.0, all its functionality moved to `camel-quarkus-core`.
 In addition to support configuring Camel through properties, `camel-quarkus-core` allows you to use conventions to configure the Camel behavior.
 For example, if there is a single `ExchangeFormatter` instance in the CDI container, then it will automatically wire that bean to the `LogComponent`.
 
-
```

Comment:
@ppalaga I hope it will complete the great work done about documentation. I think this section is not needed there as it has its own file now.